typescript/cosmos-sdk/src/client/hyperlane/interchain_security/aminomessages.ts

```diff
@@ -0,0 +1,113 @@
+import { AminoMsg } from '@cosmjs/amino';
+
+import {
+  MsgAnnounceValidator,
+  MsgCreateMerkleRootMultisigIsm,
+  MsgCreateMessageIdMultisigIsm,
+  MsgCreateNoopIsm,
+} from '../../../types/hyperlane/core/interchain_security/v1/tx.js';
+
+export interface AminoMsgAnnounceValidator extends AminoMsg {
+  readonly type: 'hyperlane/v1/MsgAnnounceValidator';
```

Comment:
Can the values of the `type` field for each one of the types in this file be part of an enum? I see that the same string is used in 2 places, so one unfamiliar with the code might change it in one place but not in the other. Example:

```typescript
enum MsgType {
 AnnounceValidator = "hyperlane/v1/MsgAnnounceValidator",
 MsgCreateMerkleRootMultisigIsm = "hyperlane/v1/MsgCreateMerkleRootMultisigIsm",
 ...
}

I'd this to be done also for the other hardcoded strings that are used in more than once